hw/mcu/sifive/src/ext/freedom-e-sdk_3235929/bsp/env/encoding.h

```diff
@@ -0,0 +1,1313 @@
+// See LICENSE for license details.
```

Comment:
Do you have a pointer to the LICENSE file which is referenced here? I think we'll want that as part of this SDK.